server/controllers/items/by_bbox.ts

```diff
@@ -0,0 +1,29 @@
+import { property } from 'lodash-es'
+import { filterMaximumItemsPerOwner } from '#controllers/items/lib/filter_maximum_items_per_owner'
+import { getItemsByUsers } from '#controllers/items/lib/get_items_by_users'
+import { getUsersByBbox } from '#controllers/user/lib/user'
+import type { SanitizedParameters } from '#types/controllers_input_sanitization_parameters'
+import type { Item } from '#types/item'
+import type { User, UserId } from '#types/user'
+
+const sanitization = {
+  bbox: {},
+  limit: {
+    default: 15,
+  },
+  lang: {
+    default: 'en',
+  },
+}
+
+async function controller (params: SanitizedParameters) {
+  const { bbox, limit, lang } = params
+  const foundUsers: User[] = await getUsersByBbox(bbox)
+  const usersIds: UserId[] = foundUsers.map(property('_id'))
```

Comment:
Without a limit on the bbox size, this could end up being a lof of users. The bbox is constrained by the client, but I don't think it would hurt to `.slice(0, 500)`, given that this endpoint also slices the returned items (not aiming for exhaustivity). A more sophisticated version could sort the users before slicing (for instance on the `user.updated` attribute to prioritize users more likely to be active users) .

Reply:
Done in 2b21d52b2